gson/src/main/java/com/google/gson/internal/ConstructorConstructor.java

```diff
@@ -40,13 +40,16 @@
 
 import com.google.gson.InstanceCreator;
 import com.google.gson.JsonIOException;
+import com.google.gson.reflect.ReflectionAccessUtils;
+import com.google.gson.reflect.ReflectionAccessor;
 import com.google.gson.reflect.TypeToken;
 
 /**
  * Returns a function that can construct an instance of a requested type.
  */
 public final class ConstructorConstructor {
   private final Map<Type, InstanceCreator<?>> instanceCreators;
+  private final static ReflectionAccessor accessor = ReflectionAccessUtils.getReflectionAccessor();
```

Comment:
This doesn't need to be static. There is just one instance of ConstructorConstructor in Gson